prov/rxd/src/rxd_cq.c

```diff
@@ -329,8 +343,12 @@ void rxd_progress_tx_list(struct rxd_ep *ep, struct rxd_peer *peer)
 
 static void rxd_update_peer(struct rxd_ep *ep, fi_addr_t peer, fi_addr_t peer_addr)
```

Comment:
Same as above - I know you didn't write this but let's update to rxd_peer instead of peer for consistency